File: c/WesleySafadaoTest.c
#include <stdio.h>
#include "WesleySafadao.h"

int main(){
  float *a = getCalc(30, 10, 94);
  printf("Você é %0.2f%% safado e %0.2f%% anjo \n", a[0], a[1]);
  return 0;
}

File: ocaml/WesleySafadaoTest.ml
#use "WesleySafadao.ml";;

let main = 
    let (year, month, day) = (28, 12, 81) in
        print_string "Você é ";
        print_int ( getPercentualSafadeza (day, month, year) ) ;
        print_string "% safado e ";
        print_int ( getPercentualAnjo (day, month, year) );
        print_endline "% anjo"
    
let _ = main
